00 Boilerplate/readme.md

```diff
@@ -167,15 +167,14 @@ Create a file named `webpack.config.js` in the root directory with the following
 
  ```javascript
  var path = require('path');
```

Comment:
Seems like we need to change all code by [this one](https://github.com/Lemoncode/redux-by-sample/blob/master/00%20Boilerplate/webpack.config.js), isn't it?

Reply:
Not all code. Just a few changes around the "ExtractTextPlugin". I made another pull request with my existing webconfig.dev.js. Most of the things that show up as "diff's" in that PR seem to be space/indenting issues. Please have a look.

Reply:
[See](https://github.com/Lemoncode/redux-by-sample/pull/54#issuecomment-336428627)